you-should-use.plugin.zsh

```diff
@@ -60,18 +60,21 @@ function _check_ysu_hardcore() {
 function _check_git_aliases() {
   if [[ "$1" = "git "* ]]; then
       local found=false
-      local tokens
+      local cmd
       local k
       local v
-      git config --get-regexp "^alias\..+$" | sort | while read entry; do
-        tokens=("${(@s/ /)entry}")
-        k="${tokens[1]#alias.}"
-        v="${tokens[2]}"
-
-        if [[ "$2" = "git $v" || "$2" = "git $v "* ]]; then
-          ysu_message "git alias" "$v" "git $k"
-          found=true
-        fi
+      git config --get-regexp "^alias\..+$" | sort | while read k v; do
+        k="${k#alias.}"
+        cmd=( ${=2##git} )
```

Comment:
I'm not familiar with this `=2` notation - would you mind explaining it to me?

Reply:
`=` before a variable name causes it to be split (like bash does with unquoted variables), so in this case it creates separate array entries. Without the `=`, this would result in a one-element-array.